modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -493,6 +493,34 @@ $ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
 CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ----
 
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
+== Running in single node cluster
+
+Running tests on a single node cluster causes the following limitations to be imposed:
+
+* Longer timeouts for certain tests, including SR-IOV and SCTP tests
+* Tests requiring master/worker nodes are skipped
+
+Longer timeouts concern SR-IOV and SCTP tests. Reconfiguration requiring node reboots cause a reboot of the entire environment, including the OpenShift control plane, and therefore takes longer to complete. All PTP tests requiring a master and worker node are skipped. No additional configuration is needed because the tests check for the number of nodes at startup and adjust test behavior accordingly.
```

Comment:
Same question about whether we can use "control plane" instead of "master" - both here and a few more instances in this PR.

Reply:
Not sure if it's just me, but I don't know what the sentence "Longer timeouts concern SR-IOV and SCTP tests." is trying to say.